### PR TITLE
ROASTER-118: Adding location functions for java types

### PR DESCRIPTION
**Pull request by kaijMueller**
Hi,
I added location functions for java types.

BR,
Kai

### CODE REVIEW

File: tests/src/test/java/org/jboss/forge/test/roaster/model/JavaPackageInfoTest.java [hunk 1]
```diff
@@ -33,6 +34,16 @@ public void testCanCreatePackageInfo()
       Assert.assertEquals("org.jboss.forge.roaster", packageInfo.getPackage());
       Assert.assertEquals("package-info", packageInfo.getName());
    }
+   @Test
+   public void testLocation() {
+      JavaPackageInfoSource packageInfo = Roaster.create(JavaPackageInfoSource.class);
+      packageInfo.setPackage("abc");
+      Assert.assertThat(packageInfo.getStartPosition(), is(0));
+      System.out.println(packageInfo);
```
Comment (gastaldi) [inline tests/src/test/java/org/jboss/forge/test/roaster/model/JavaPackageInfoTest.java]: Mind removing this System.out?

Comment (kaijMueller) [reply tests/src/test/java/org/jboss/forge/test/roaster/model/JavaPackageInfoTest.java]: ups, thank you. I remove this

**Comment by gastaldi** [comment]
Don't forget to rebase with `master`

File: tests/src/test/java/org/jboss/forge/test/roaster/model/NestedClassTest.java [hunk 2]
```diff
@@ -58,6 +61,31 @@ public void testGetNestedClasses()
...
+      Assert.assertThat(inner3.getColumnNumber(), is(inner1.getColumnNumber() + innerClass1.length()));
+
+      Assert.assertThat(javaClass.getEndPosition(), is(pckage.length() + outerClass.length() + innerClass1.length()
+               + innerClass2.length() + innerClass3.length() + 2));
+      Assert.assertThat(inner1.getEndPosition(), is(pckage.length() + outerClass.length() + innerClass1.length() + innerClass3.length() + 1));
+      Assert.assertThat(inner2.getEndPosition(),
+               is(javaClass.getEndPosition()-1));
+      Assert.assertThat(inner3.getEndPosition(), is(inner1.getEndPosition() - 1));
+      
+      //no line separators -> so line numer 1
```
Comment (gastaldi) [inline tests/src/test/java/org/jboss/forge/test/roaster/model/NestedClassTest.java]:
```suggestion
      //no line separators -> so line number 1
```

**Pull request merged**